pages_/my-shop.tsx

```diff
@@ -265,15 +269,15 @@ const MyShop = ({ isLoggedIn, shopId, encodedShopId }: Props) => {
 export const getServerSideProps: GetServerSideProps = async (context) => {
   const token = getToken(context);
 
-  if (!token || !token.shopId) {
-    return { props: { isLoggedIn: false } };
+  if (!token || !token?.isValid || !token.clientId) {
```

Comment:
`!token?.isValid` isn't the `getToken` function already doing this? And why the `!token.clientId` condition?

Reply:
yes. and it should be shopId instead. I was thinking something, then I realize it was not needed and I forget to remove the code.